[2025-07-22T03:48:15.641+0000] {scheduler_job_runner.py:769} ERROR - Executor reports task instance <TaskInstance: fetch_weather_foreacst_dag.create_staging_table scheduled__2025-07-21T18:00:00+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
